extensions/cactus-plugin-htlc-coordinator-besu/package.json

```diff
@@ -43,9 +43,9 @@
     "dist/*"
   ],
   "scripts": {
-    "codegen": "run-p 'codegen:*'",
-    "codegen:openapi": "run-p generate-sdk",
-    "generate-sdk": "run-p generate-sdk:*",
+    "codegen": "yarn run codegen:openapi",
```

Comment:
@zondervancalvez Same question here as in my other comment. Why run only a specific script instead of the pattern?

Reply:
After few tests, we revert this to original and added minor changes. We added single quote to `generate-sdk:*`